content/applications/marketing/marketing_automation/getting_started/first_campaign.rst

```diff
@@ -1,59 +1,108 @@
-====================
-Marketing Automation
-====================
+==============================
+Marketing automation campaigns
+==============================
 
-The Odoo Marketing Automation app automates a variety of marketing tasks by combining specific
+The Odoo *Marketing Automation* app automates a variety of marketing tasks by combining specific
 rules and filters to generate timed actions. Instead of manually having to build each stage of a
-campaign (such as a series of timed massmails), the Marketing Automation app allows marketers to
-build the entire campaign, and all of its stages, in one place on one dashboard.
+campaign (like a series of timed massmails), the *Marketing Automation* app allows marketers to
+build the entire campaign, and all of its stages, in one place -- on one dashboard.
 
-Create a campaign
-=================
+Campaign configuration
+======================
 
-To create a new automated marketing campaign, open the :guilabel:`Marketing Automation` app and
-click :guilabel:`Create`. On the :guilabel:`Campaign` page, there are the following smart buttons
-and fields:
+To create a new automated marketing campaign, navigate to :menuselection:`Marketing Automation app
+--> New` to reveal a blank campaign detail form.
 
-.. image:: first_campaign/marketing-template-sample.png
+.. image:: first_campaign/blank-marketing-campaign-form.png
    :align: center
-   :alt: A dashboard showing the creation of a new marketing automation campaign in Odoo.
-
-- :guilabel:`Templates`: represents the number of pre-configured mail templates being used in this
-  particular campaign. (Templates can always be created on-the-fly as well).
-- :guilabel:`SMS`: represents the number of personalized SMS messages connected to this campaign.
-- :guilabel:`Clicks`: represents the number of times attached links have been clicked by recipients
-  of this campaign.
-- :guilabel:`Participants`: represents the number of contacts that have directly participated in
-  this campaign.
-- :guilabel:`Target`: this field is a drop-down menu to choose which model is targeted by this
-  campaign (i.e., by Contacts, Sales Order, Lead/Opportunity, etc.).
-
-Campaign filters
-================
-
-To add a filter to the target audience, click :guilabel:`Add Filter`, and a node field will
-appear. In the node field, a custom equation can be configured for Odoo to use when filtering who
-to include (and exclude) in this specific marketing campaign.
+   :alt: A blank marketing automation campaign form in Odoo Marketing Automation application.
 
-.. image:: first_campaign/filter-node.png
-   :align: center
-   :alt: A filter node in Odoo Marketing Automation.
+After entering a name for the marketing campaign, configure the target audience in the remaining
+fields.
```

Comment:
I know that you detail how to do this in the paragraphs below, but I think it would be helpful if there were 1-2 sentences explaining exactly what it means/what it does when you choose an audience for the campaign. Why is it significant?